app/wasm.go

```diff
@@ -1,9 +1,30 @@
 package app
 
-import "strings"
+import (
+	"strings"
+
+	wasmkeeper "github.com/CosmWasm/wasmd/x/wasm/keeper"
+	tokenfactorytypes "github.com/public-awesome/stargaze/v13/x/tokenfactory/types"
+)
 
 var wasmCapabilities = []string{
-	"iterator", "staking", "stargate", "stargaze", "cosmwasm_1_1", "token_factory",
+	"iterator",
+	"staking",
+	"stargate",
+	"stargaze",
+	"cosmwasm_1_1",
+	"cosmwasm_1_2",
+	"cosmwasm_1_3",
+	"cosmwasm_1_4",
```

Comment:
pending `cosmwasm_1_5`

Reply:
Couldnt find any docs on `cosmwasm_1_5` capabilities https://github.com/CosmWasm/cosmwasm/blob/v1.5.0/docs/CAPABILITIES-BUILT-IN.md 🤔